src/pages/home/home.ts

```diff
@@ -440,8 +510,11 @@ export class HomePage {
               this.isInGroup = false;
               this.isSpaceCreated = false;
               this.selectedLocation = null;
-              for (var i = 1; i < this.droppedMarkers.length; i++) {
-                this.droppedMarkers[i].remove();
+              for (var i = this.droppedMarkers.length - 1; i >= 0; i--) {
+                if (this.droppedMarkers[i].title != 'Current Location') {
```

Comment:
Remember to use `===` or `!==`

Reply:
Already used the filter thing that you suggested!